mhs/inbound/inbound/request/tests/test_handler.py

```diff
@@ -107,32 +109,37 @@ def test_correct_workflow(self):
 
         self.assertEqual(ack_response.code, 200)
 
-    @unittest.mock.patch.object(message_utilities.MessageUtilities, "get_timestamp")
-    @unittest.mock.patch.object(message_utilities.MessageUtilities, "get_uuid")
-    def test_workflow_throws_exception(self, mock_get_uuid, mock_get_timestamp):
+    def test_workflow_throws_exception(self):
         self.mock_workflow.handle_inbound_message.side_effect = Exception("what a failure")
-        mock_get_uuid.return_value = "5BB171D4-53B2-4986-90CF-428BE6D157F5"
-        mock_get_timestamp.return_value = "2012-03-15T06:51:08Z"
         request_body = file_utilities.FileUtilities.get_file_string(str(self.message_dir / REQUEST_FILE))
 
         response = self.fetch("/", method="POST", body=request_body, headers=CONTENT_TYPE_HEADERS)
 
         self.assertEqual(response.code, 500)
-        expected = b'<html><title>500: Exception in workflow</title><body>500: ' \
-                   b'Exception in workflow</body></html>'.decode('utf-8')
+        expected = '<html><title>500: Exception in workflow</title><body>500: Exception in workflow</body></html>'
         self.assertEqual(response.body.decode('utf-8'), expected)
 
-    @unittest.mock.patch.object(message_utilities.MessageUtilities, "get_timestamp")
-    @unittest.mock.patch.object(message_utilities.MessageUtilities, "get_uuid")
-    def test_no_reference_to_id(self,  mock_get_uuid, mock_get_timestamp):
-        mock_get_uuid.return_value = "5BB171D4-53B2-4986-90CF-428BE6D157F5"
-        mock_get_timestamp.return_value = "2012-03-15T06:51:08Z"
+    def test_no_reference_to_id(self):
         request_body = file_utilities.FileUtilities.get_file_string(str(self.message_dir / NO_REF_FILE))
 
         ack_response = self.fetch("/", method="POST", body=request_body, headers=CONTENT_TYPE_HEADERS)
 
         self.assertEqual(ack_response.code, 500)
 
+    def test_logging_context_variables_are_set(self):
+        log.correlation_id = unittest.mock.MagicMock()
+        log.inbound_message_id = unittest.mock.MagicMock()
```

Comment:
Modify these variables using the `patch` decorator, so that they get reset afterwards, so that there's no risk of any other tests being affected.